packages/suite/src/components/wallet/AccountsMenu/AccountGroup.tsx

```diff
@@ -56,6 +56,8 @@ const getGroupLabel = (type: AccountGroupProps['type']) => {
     if (type === 'taproot') return 'TR_TAPROOT_ACCOUNTS';
     if (type === 'legacy') return 'TR_LEGACY_ACCOUNTS';
     if (type === 'ledger') return 'TR_CARDANO_LEDGER_ACCOUNTS';
+    if (type === 'bip44Change') return 'TR_SOLANA_BIP44_CHANGE_ACCOUNTS';
```

Comment:
Does solana even have change accounts? They don't really make sense to me mentallly since solana uses the account model, not UTXO. Where did you find info about this?

Reply:
Jop it does, 
https://help.phantom.app/hc/en-us/articles/12988493966227-What-derivation-paths-does-Phantom-wallet-support-